librz/include/rz_util/ht_inc.h

```diff
@@ -81,8 +81,21 @@
 #define HT_NULL_VALUE  0
 #endif
 
-#ifndef HT_STR_OPTION_DEFINED
-#define HT_STR_OPTION_DEFINED
+#ifndef HT_ENUM_DEFINED
+#define HT_ENUM_DEFINED
+/**
+ * Return codes for insert/update methods
+ * code < 0 <--> code == HT_RC_ERROR
+ * code >= 0 <--> code != HT_RC_ERROR
+ * code > 0 <--> code == HT_RC_INSERTED || code == HT_RC_UPDATED
+ */
+typedef enum {
+	HT_RC_ERROR = -1, ///< Error (out of memory)
+	HT_RC_EXISTING = 0, ///< Existing KV prevented an insertion
+	HT_RC_INSERTED = 1, ///< New KV was inserted during insert/update operation
+	HT_RC_UPDATED = 2, ///< Existing KV was updated during update operation
+} HtRetCode;
```

Comment:
Following the PR changes, these are mostly used as return values. it would be helpful to convert those `int` into `HtRetCode`

Reply:
Ok maybe I misunderstood the point about SWIG in `DEVELOPERS.md`

Reply:
Probably it needs to be updated since we use the new bindings code for this. @XVilka am i right or i misunderstood this ?